cmd/slinky/main.go

```diff
@@ -66,10 +69,24 @@ var (
 
 func init() {
 	rootCmd.Flags().StringVarP(
-		&oracleCfgPath,
+		&marketMapProvider,
+		"marketmap-provider",
+		"",
+		marketmap.Name,
+		"MarketMap provider to use (marketmap_api, dydx_api, dydx_research_api).",
```

Comment:
Do you think it's a good idea to leave `dydx_research_api` undocumented? Seems like something we don't really want anyone else to use.

Reply:
THat makes sense